src/llfs/page_allocator.hpp

```diff
@@ -111,14 +112,14 @@ class PageAllocator
   // TODO [tastolfi 2023-03-22] deprecate public usage of this function!
   //
   template <typename T>
-  StatusOr<slot_offset_type> update(const T& event);
+  StatusOr<slot_offset_type> update(T& event);
 
   // Updates the index synchronously by applying the specified event.  Must be one of the event
   // types enumerated in page_device_event_types.hpp.
   // TODO [tastolfi 2023-03-22] deprecate public usage of this function!
   //
   template <typename T>
-  StatusOr<slot_offset_type> update_sync(const T& event);
+  StatusOr<slot_offset_type> update_sync(T& event);
 
   // Block until updates have caught up with the specified slot number.
   //
```

Comment:
Add comment to doc of `update_page_ref_counts`: only the ***last*** delta for a given `page_id` in the sequence (`ref_count_updates`) will be applied, earlier ones are ignored.

Reply:
Added.